### PR TITLE
Poetry ify

### PR DESCRIPTION
**Pull request by m1yag1**

### CODE REVIEW

**Comment by TylerZeroMaster** [comment]
This branch makes pipemgr an installable package (thanks to @m1yag1) and updates the output paths so that osbooks.yml is stored and ~/.pipemgr and the generated pipeline will output to ./sync-osbooks.yml by default.

**Comment by m1yag1** [comment]
I can't approve this PR since I'm one of the authors. It looks like someone else on @openstax/ce-tech will need to review other than myself or @TylerZeroMaster .

File: pipemgr/src/osbook_utils.py [hunk 1]
```diff
@@ -6,15 +6,16 @@
...
 class OSBook:
     def __init__(self, book_repo: str, archive_server: str):
         self.book_repo = book_repo
         self.archive_server = archive_server
 
     def __eq__(self, o: object) -> bool:
         return (isinstance(o, OSBook)
-            and o.book_repo == self.book_repo
-            and o.archive_server == self.archive_server)
+                and o.book_repo == self.book_repo
```
Comment (ckline-tryptic) [inline pipemgr/src/osbook_utils.py:17]: I take it the extra indent is intentional?

**Review by ckline-tryptic — APPROVED** [review]

File: pipemgr/src/osbook_utils.py (hunk 1 above)
Comment (TylerZeroMaster) [reply pipemgr/src/osbook_utils.py:17]: That and many other whitespace changes were added by automatic formatting. It's within parenthesis too, so indent doesn't really matter.

**Pull request merged**